src/data/balance/starters.ts

```diff
@@ -0,0 +1,1224 @@
+import { Abilities } from "#enums/abilities";
+import { Species } from "#enums/species";
+
+export const POKERUS_STARTER_COUNT = 5;
+
+export function getStarterValueFriendshipCap(value: integer): integer {
+  switch (value) {
```

Comment:
```suggestion
export function getStarterValueFriendshipCap(starterCost: integer): integer {
  switch (starterCost) {
```

For clarity. Doc would be nice as well

Reply:
```suggestion
export function getStarterValueFriendshipCap(starterCost: number): integer {
  switch (starterCost) {
```
Adding: no more `integer`